### ISSUE TITLE
SelectListView with GridView not working properly in ActionPage

### ISSUE THREAD

**Issue by hem-bhagat-tark** [HomeSeer/Plugin-SDK#314]
#### OS
Windows
#### HS Version
v4.2.14
#### PSDK Version
v1.4.2.0
#### Language
C#
#### IDE
Visual Studio 2022
#### Dev OS
Windows
### Plugin
I am developing a plugin

### Description
I am creating an ActionType from the plugin I am developing. In my ActionType.OnNewAction I have initialized the page as follows,
```c#
gridRow.AddItem(...select list view 1)
gridView.AddRow(gridRow)
pageFactory.WithView(gridView)
ConfigPage = pageFactory.Page
```
In ActionType.OnConfigItemUpdate when the select list view 1 changes, I want to add two new select list views in the same grid row, so I have done as follows

```c#
// Again creating an already existing select list view
gridRow.AddItem(...select list view 1)
// Adding new select list views
gridRow.AddItem(...select list view 2)
gridRow.AddItem(...select list view 3)
gridView.AddRow(gridRow)
pageFactory.WithView(gridView)
ConfigPage = pageFactory.Page
return true
```

In ActionType.OnConfigItemUpdate when the select list view 2 or select list view 3 changes, I have just returned true.

**This is where I am facing the issue. The data of select list view 2 and 3 is saved and we can check that using <select list view 2>.Selection. But the select list view 2 and select list view 3 does not reflect this in UI. That is even though they have a value selected, the selected value is not shown in UI.**

What am I doing wrong here?
### Expected Behavior
The UI should reflect the selected value in select list 2 and select list 3.

### Steps to Reproduce
_Same as description_

### Screenshots
_Not applicable_

### Logs
_Not applicable_

**Comment by spudwebb** [comment]
I can reproduce the problem, and I think I know what is causing it.
Thanks for reporting.

Tracked as PSDK-291

**Comment by hem-bhagat-tark** [comment]
@spudwebb I have updated to HS4 4.2.15.0 and now data is not getting saved for Event Action Pages created using Plugin SDK. I debugged and found that the `OnConfigItemUpdate(AbstractView configViewChange)` method is not being called when there is a change in the Event Action Page.

Please let me know if you can reproduce this. Should I create a new issue for this?

**Comment by spudwebb** [comment]
Can you make sure it isn't the same problem as https://github.com/HomeSeer/Plugin-SDK/issues/317 ?
If it is a different problem then create a new issue.

**Comment by hem-bhagat-tark** [comment]
Yes, it is the same as #317. No need to create a new one. Currently working in firefox.

**Comment by alexbk66** [comment]
@hem-bhagat-tark 
https://docs.homeseer.com/products/software/hs4-smart-home-software/hs4-release-notes-downloads/hs4-release-4-2-17-0

**Comment by hem-bhagat-tark** [comment]
The Releases section is still showing v1.4.2 as the latest release.

![image](https://user-images.githubusercontent.com/104339756/222345505-81e66632-41fc-4dc7-9c8f-f80606169ce0.png)
